src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -100,7 +100,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <StaticSslLibs Include="-Wl,-Bdynamic" Condition="'$(StaticExecutable)' != 'true'" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetOS)' == 'osx'">
+    <ItemGroup Condition="'$(_IsApplePlatform)' == 'true'">
```

Comment:
`<NativeFramework Include="GSS" />` in the group won't work for tvOS[simulator] IIRC.

Reply:
Should it be excluded for tvOS[simulator]?

Reply:
I think so. The library builds don't reference the GSS API on tvOS builds.